cinema/views.py

```diff
@@ -34,25 +45,71 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(query_string):
+        return [int(str_id) for str_id in query_string.split(",")]
```

Comment:
The `_params_to_ints` method should include error handling to manage cases where the query string contains non-integer values. Consider using a try-except block to catch `ValueError` exceptions.